src/LiveSplit.WorldRecord/UI/Components/WorldRecordComponent.cs

```diff
@@ -58,7 +58,7 @@ public WorldRecordComponent(LiveSplitState state)
 
         RefreshInterval = TimeSpan.FromMinutes(5);
         Cache = new GraphicsCache();
-        TimeFormatter = new GeneralTimeFormatter()
+        WRTimeFormatter = new RegularTimeFormatter()
```

Comment:
Why are you changing this to `RegularTimeFormatter`? That already inherits from `GeneralTimeFormatter`

Reply:
I think I couldn't edit the accuracy after it was created.

Reply:
Not sure why `RegularTimeFormatter` would make a difference with that?

Reply:
Issue was the ITimeFormatter; I've changed both to general type now